test/Table_Tests/src/Common_Table_Operations/Column_Operations_Spec.enso

```diff
@@ -1734,6 +1761,36 @@ add_specs suite_builder setup =
             t = table_builder [["x", ["1", "2", "3"]]]
             t.at "x" . const (t.at "x") . should_fail_with Illegal_Argument
 
+    if setup.is_database then
+        suite_builder.group prefix+"(Column_Operations_Spec) Column.let" group_builder->
+            connection_data = Connection_Data.setup create_connection_fn
+
+            group_builder.teardown <|
+                connection_data.teardown
+
+            if connection_data.connection.dialect.supports_nested_with_clause then
```

Comment:
@radeusgd I'm using the dialect for this. It could also be a test flag, but then we would have this flag in both the test config and the dialect, which seems redundant.

Reply:
👍 that seems very reasonable